src/components/Tree/TreeItem.tsx

```diff
@@ -9,33 +19,66 @@ import { DraggableItem, DropZonePosition } from '@utilities/dnd';
 import { DropZone } from '@components/DropZone';
 import type { TreeItemProps } from '@components/Tree/types';
 import { useTreeContext } from '@components/Tree/TreeContext';
+import { FOCUS_VISIBLE_STYLE } from '@utilities/focusStyle';
 
-import { useDraggableEnhancedChildren } from './hooks/useDraggableEnhancedChildren';
+import { useDraggableEnhancedChildren } from './hooks';
+import { getAcceptTypes, getItemPositionInParent, getNextItemToFocus, getPreviousItemToFocus } from './helpers';
+import { cloneThroughFragments, flattenChildren } from './utils';
 
 const DRAGGING_OPACITY = 0.4;
 const DEFAULT_OPACITY = 1;
 
 export const TreeItem = ({
     id,
-    sort,
+    parentId,
```

Comment:
why does TreeItem have a `parentId` prop? Should that be inside treeState?

Reply:
What is level used for?

Reply:
> why does TreeItem have a parentId prop? Should that be inside treeState?
Passed by its parent when rendering happens

> What is level used for?
[aria-level](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-level)